scripts/build/osDistro.sh

```diff
@@ -22,8 +22,10 @@ ubuntu_distro=$(grep DISTRIB_DESCRIPTION= /etc/lsb-release 2>/dev/null | cut -d\
 debian_distro=$(cat /etc/debian_version 2> /dev/null)
 suse_distro=$(cat /etc/SuSE-release 2> /dev/null | grep SUSE | cut -d ' ' -f 1-2 | sed 's/ /_/g')
 centos_distro=$(cat /etc/redhat-release 2> /dev/null | awk '{print $3}')
+if [ $centos_distro == "release" ]; then
```

Comment:
Good fix. However, I'd suggest to add some explanation about this, as it may be a bit criptic. For example, just above this line:

```
# In some cases (e.g. CentOS 7.x) we have found that the /etc/redhat-release content use the following pattern:
#
# CentOS Linux release 7.2.1511 (Core) 
# 
# Instead the one in previous versions:
#
# CentOS release 6.6 (Final)
#
# The check in the next line ensure that the distro version is correctly detected in any case.
```

Or a similar wording.